examples/jsm/postprocessing/TAARenderPass.js

```diff
@@ -133,7 +133,6 @@ class TAARenderPass extends SSAARenderPass {
 			this.copyUniforms[ 'opacity' ].value = 1.0 - accumulationWeight;
 			this.copyUniforms[ 'tDiffuse' ].value = this.holdRenderTarget.texture;
 			renderer.setRenderTarget( writeBuffer );
-			if ( accumulationWeight === 0 ) renderer.clear();
```

Comment:
The problem was this clear. When it happens it deleted the contents of the previous render which explains the flash at the beginning of the accumulation phase. The write buffer has already been cleared by the previous call of `this.fsQuad.render( renderer );` so it should be safe to delete it.